### PR TITLE
Proposed Constitution Updates

### PR DESCRIPTION
**Pull request by ifgc**
These are a series of proposals to update the role of the strategy lead and its relationship to other positions. It was approved by a 6/7 majority leads vote with on abstain.

### CODE REVIEW

**Review by johnluetke — COMMENTED** [review]
Per slack, 6/7 leads voted to approve with 1 abstaining. Can those votes be recorded here for reference?

@ishanthecoder there are a few formatting/grammar errors ~~here that I would like to run by you.~~ that I left comments for.

Finally, @organizefish must approve and merge all changes the the constitution

File: Club Constitution.md [hunk 1]
```diff
@@ -185,23 +185,24 @@ The following leads are responsible for maintaining the following roles.
...
-  * Research Strategies
-  * Coordinate with other teams at competitions
-  * Have the strategy lead (whether elected or appointed) be responsible for competitive analysis (managing scouting/drive team)
-  * Have the strategy lead (whether elected or appointed) advise design on game analysis
-  * Require SL to help lead kickoff discussions regarding game analysis and rule book understanding
-  * The strategy lead needs to be responsible for updating the team about the status of strategy and updates to the rule book
-  * Have strategy training on game analysis and competitive analysis and driving practice
-  * SL is responsible for ensuring we have a drive team and head scout
-  * SL should be in charge of reading the rule book and being an expert on it
+  *Must research and/or be knowledgeable in the areas of Strategic Design, Scouting, Match Strategies, and Robot Control & Operation, or similar
```
Comment (johnluetke) [inline Club Constitution.md]: - Space after `*`
- `and/or` should be just `and`

File: Club Constitution.md [hunk 2]
```diff
@@ -185,23 +185,24 @@ The following leads are responsible for maintaining the following roles.
...
+ * Lead the Strategy Department by teaching the curriculum, working to recruit and retain members, and attending leads meetings,
+  * Organise and lead the team in through the Game Analysis phase of Strategic Design on Kickoff
+  * Must lead the team through the Cost-Benefit Analysis phase of Strategic Design
+  *Must compile the ultimate goal for the season, general game strategy and robot priorities into a draft of the Design Platform with help from the Strategy Department and Design Specialist.
```
Comment (johnluetke) [inline Club Constitution.md]: - Space after `*`

File: Club Constitution.md [hunk 3]
```diff
@@ -185,23 +185,24 @@ The following leads are responsible for maintaining the following roles.
...
+  *Responsible for editing and presenting the Design Platform to the Design Specialist for review, which will be frozen upon the signature of both leads.
```
Comment (johnluetke) [inline Club Constitution.md]: - Space after `*`

**Review by johnluetke — COMMENTED** [review]
Are these responsibilities ordered in any way? Should they be?

**Review by johnluetke — APPROVED** [review]

**Review by organizefish — APPROVED** [review]

**Pull request merged**